standalone/runtime/src/lib.rs

```diff
@@ -457,6 +458,15 @@ impl_runtime_apis! {
 		fn set_dkg_pub_key(key: Vec<u8>) -> () {
 
 		}
+
+		fn get_unsigned_proposals() -> Vec<(ProposalNonce, ProposalType)> {
+			// DKGProposalHandler::get_unsigned_proposals()
+			Vec::new()
+		}
+
+		fn add_signed_proposal(proposal: ProposalType) {
```

Comment:
We should remove this. Runtime APIs are just for making queries to state not for making state changes.